app/controllers/nonplayers_controller.rb

```diff
@@ -46,10 +38,7 @@ def create
   # PATCH/PUT /nonplayers/1 or /nonplayers/1.json
   def update
     respond_to do |format|
-      new_params = nonplayer_params
-      new_params[:item_to_offer] = Item.find(Integer(nonplayer_params['item_to_offer']))
-      new_params[:item_to_accept] = Item.find(Integer(nonplayer_params['item_to_accept']))
-      if @nonplayer.update(new_params)
+      if @nonplayer.update(nonplayer_params)
         format.html { redirect_to nonplayer_url(@nonplayer), notice: "#{@nonplayer.name} was successfully updated." }
```

Comment:
These two lines in the cases are pretty similar for both update and create. Maybe we can abstract them into helper functions since rubycop is complaining about duplication?